qiskit/visualization/pulse_v2/types.py

```diff
@@ -33,11 +34,11 @@ class PhaseFreqTuple(NamedTuple):
 PhaseFreqTuple.freq.__doc__ = "Frequency value in Hz."
 
 
-PulseInstruction = NamedTuple(
+PulseInstruction = NamedTuple(  # type: ignore[name-match]
```

Comment:
I would leave these `name-match`. They are indeed an inconsistence that it might be worthy to take a look at some point.